### ISSUE TITLE
Model validation accuracy collapsed

### ISSUE THREAD

**Issue by Thunfischpirat** [Thunfischpirat/SpokenDigits#20]
Somehow the neural network model's performance during training doesn't surpass 10% anymore. Why is that the case?
